chambersf/encounters/mpg_corruption.lua

```diff
@@ -245,8 +250,13 @@ function Huhn_Signal(e)
     eq.depop_with_timer();
 
     -- Set a Lockout 
+    local dz = eq.get_expedition()
+    if dz.valid then
+      dz:AddLockoutDuration("Replay Timer", eq.seconds("5d")) -- 5 days + current timer (max 123 hours)
```

Comment:
Is "Replay Timer" a fixed constant string used for writing lockouts in general? Would it be anything else?

Reply:
I was actually looking at this one today as well. This is a relatively new api only added in the last few weeks after seeing how live did some lockouts. "Replay Timer" is a constant string used for non-event based lockouts and I should have added an AddReplayLockoutDuration for this. I may also just export a GetReplayTimerName method for things like this